hibernate-core/src/main/java/org/hibernate/loader/ast/internal/SingleIdLoadPlan.java

```diff
@@ -73,21 +73,22 @@ T load(
 			LockOptions lockOptions,
 			Boolean readOnly,
 			SharedSessionContractImplementor session) {
-		return load( restrictedValue, lockOptions, null, readOnly, session );
+		return load( restrictedValue, lockOptions, null, readOnly,false, session );
 	}
 
 	T load(
 			Object restrictedValue,
 			LockOptions lockOptions,
 			SharedSessionContractImplementor session) {
-		return load( restrictedValue, lockOptions, null, null, session );
+		return load( restrictedValue, lockOptions, null, null,false, session );
 	}
 
 	T load(
 			Object restrictedValue,
 			LockOptions lockOptions,
 			Object entityInstance,
 			Boolean readOnly,
+			Boolean singleResultExpected,
```

Comment:
I was actually thinking about pulling the `UniqueSemantics` up to here to avoid all the boolean stuff down the line. Wdyt?

Reply:
done, Thanks @beikov